.buildkite/pipeline_perf.py

```diff
@@ -7,9 +7,15 @@
 from common import COMMON_PARSER, group, overlay_dict, pipeline_to_json
 
 perf_test = {
-    "block": {
+    "virtio-block": {
```

Comment:
Just FYI: Please remember to update the step definition of the nightly performance pipeline once this PR gets merged.

Reply:
very nicely spotted, thanks!